app/src/main/java/com/nervousfish/nervousfish/cryptography/KeyGenerator.java

```diff
@@ -0,0 +1,62 @@
+package com.nervousfish.nervousfish.cryptography;
+
+import java.security.KeyFactory;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.RSAPrivateKeySpec;
+import java.security.spec.RSAPublicKeySpec;
+
+public class KeyGenerator {
+
+    /**
+     * Generates a random KeyPair with the RSA algorithm.
+     * @return a randomly generated KeyPair
+     */
+    public static KeyPair generateRandomKeyPair() {
```

Comment:
After structure is merged, these methods should not be static, but just normal methods